core/src/trezor/pin.py

```
@@ -6,84 +6,87 @@
+    global _scr, _timer
 
     if callable(keepalive_callback):
         keepalive_callback()
 
-    if message is not None:
-        text = ""
-        if message == "read fp data":
+    _scr_need_create = _scr is None
+    if _scr_need_create:
+        try:
+            _scr.subtitle
```

Ruff (0.12.2)

Found useless expression. Either assign it to a variable or remove it.

(B018)